include/deal.II/numerics/vector_tools_evaluate.h

```diff
@@ -108,6 +108,9 @@ namespace VectorTools
    *   cache, dof_handler_2, vector_2);
    * @endcode
    *
+   * @note If a point cannot be found, the result for these points will
+   *   be undefined (most probably 0).
```

Comment:
I would write this differently: 

"If RemotePointEvaluation::reinit() called by the present function does not find a point, the result for this point will be undefined (most probably 0). To make sure that all points can be evaluated by this function, the other point_values() function, for which cache has already been initialized prior to calling point_values(), should be used."

I don't know if "most probably 0" is useful information.